apps/api/src/app/events/usecases/run-job/run-job.usecase.ts

```diff
@@ -48,14 +71,16 @@ export class RunJob {
 
     await this.storageHelperService.deleteAttachments(job.payload?.attachments);
 
-    await this.queueNextJob.execute(
-      QueueNextJobCommand.create({
-        parentId: job._id,
-        environmentId: job._environmentId,
-        organizationId: job._organizationId,
-        userId: job._userId,
-      })
-    );
+    if (isDigestType) {
```

Comment:
Like mentioned above, queued if it is digest type, to have the latest information about `digest events`  during processing inside workflow service